### PR TITLE
feat: debug with selector option added

### PR DESCRIPTION
**Pull request by Bayheck**
## Purpose
The purpose of this PR is to allow passing selector to a t.debug call. Something like this t.debug(Selector). Then the selector will be shown in debug mode and highlighted.

## Approach
So overall, the main goal was to pass the selector argument from server-side debug call to client-side selector-input-contaner. 
1) src/test-run/index.ts Inside this file we have _internalExecuteCommand method. There is debug command type handling block. Inside of it we call _enqueueSetBreakpointCommand to which we pass command.selector as a second argument. This way we transfer Selector from debugCommand to breakpointCommand.
2) After the breakpointCommand is added to driverTaskQueue we can work with it on client-side.
3) On client-side we pass selector to _debugSelector  method of SelectorInputContainer class.
4) In _debugSelector method we:
           -   remove any previously highlighted elements,
           -  get Selector name and assign it to this.value for SelectorInputContainer,
           -  execute Selector to get elements for highlighting and indication,
           -  highlight and indicate elements of the Selector

## References
https://github.com/DevExpress/testcafe-private/issues/163

## Pre-Merge TODO
- [x] Write tests for your proposed changes
- [x] Make sure that existing tests do not fail

### CODE REVIEW

File: test/functional/fixtures/ui/testcafe-fixtures/selector-inspector-test.js [hunk 1]
```diff
@@ -283,3 +283,33 @@ test('should hide panel', async t => {
 
     await t.debug();
 });
+
+test('should indicate the correct number of elements matching the TestCafe selector passed in debug', async t => {
```
Comment (Aleksey28) [inline test/functional/fixtures/ui/testcafe-fixtures/selector-inspector-test.js:287]: Your tests aren't executed. You should add them [here](https://github.com/DevExpress/testcafe/blob/d55e8b8145a1d32c9665893817295f6fa70e69ad/test/functional/fixtures/ui/test.js#L45)

**Review by Aleksey28 — CHANGES_REQUESTED** [review]
Good job!  Please look at my comments

**Review by Aleksey28 — CHANGES_REQUESTED** [review]
One more thing

**Review by Aleksey28 — APPROVED** [review]
Great, merge after the tests turn green

**Comment by aleks-pro** [comment]
Please, do not merge. We need add documentation first.

**Pull request merged**